pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/DrugTypeaheadComparator.java

```diff
@@ -22,58 +22,13 @@
      */
     public DrugTypeaheadComparator(String searchTerm) {
         this.searchTerm = searchTerm.toUpperCase();
+        this.stringComparator = new StringSuggestionTypeaheadComparator(searchTerm);
     }
 
     @Override
     public int compare(Drug drug1, Drug drug2) {
```

Comment:
Refactored to use StringTypeaheadComparator and avoid code duplication.
Cancer, Drug uses exact same logic to compare & sort the suggestions